resources/[core]/customization/src/client/managers/creation-manager.ts

```diff
@@ -218,6 +222,24 @@ class CreationManager {
     this.currentState = 1;
   }
 
+  setFirstName(name: string) {
+    this.firstName = name;
+  }
+
+  setLastName(name: string) {
+    this.lastName = name;
+  }
+
+  setDateOfBirth(date: Date) {
+    this.dateOfBirth = date;
+  }
+
+  async setInformation() {
```

Comment:
You wont need to set name before customizing. There will be tabs to freely choose between the available sections you can change during creaton.